src/_index.js

```diff
@@ -40,7 +39,20 @@ class Popup{
         this.options.hasMobileLayout = this.isBooleanOptionTrue(ATTRS.mobileLayout, this.options.hasMobileLayout);
 
         // get options from JSON init
-        this.options = getOptions(this, this.options);
+        this.id = options.id || this.el.id || DEFAULTS.id;
+
+        this.options = getOptionsFromAttribute(
+            {
+                target: this.el,
+                attributeName: ATTRS.init,
+                defaultOptions: {...this.options, id: this.id},
+                numericValues: ['autoShow'],
```

Comment:
Did you check all the numeric values of the options?

Reply:
Oh, there are one more argument "showingTimes"